docs/community/casestudies.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How to Post Your Case
+sidebar_position: 6
+---
+# Case Studies 
+
+The KubeEdge Case Studies is designed to show successful cases of users who have implemented KubeEdge in their business scenarios. Here, you can share your successful cases with other users, so that they can learn how to use KubeEdge in real industry settings. We also hope that you can gain inspiration from browsing other successful case studies and apply them to your own projects.
+
+This guide aims to help you understand how to submit your successful case studies to the community case center and provide guidelines for writing case studies.
+
+## Writing Guidelines
+
+To ensure the quality and readability of case studies, we encourage you to provide as much detail as possible about the business scenario, technical architecture, final outcome and etc. You can refer to existing case studies as a reference and please use `Markdown` format when writing the document.
+
+1.  At the beginning of the document, please include the following information. This information will be displayed on the KubeEdge official website's case studies homepage in a card format, allowing users to quickly understand the general content of the case.
+
+```
+title: Company Name
+subtitle：(Optional) Please use keywords to describe the significant effect of the case study. (Example: 100% Performance Improvement)
+description： Briefly summarize the case study in one sentence.
+date： Date (Format: 2023-01-01)
+tags： Case tags. Please select Solution or End User based on the type of case.
+```
+
+2.  Main Content: Please provide a detailed description of the case study in the main content of the document. You may consider the following points for detailed exposition:
+
+- Case title
+- Case background
+- Application scenario
+- Challenges faced
+- Why choose KubeEdge
+- Solution and technical architecture
+- Final outcome and realized value
+- Future plans
+
+# Submit Your Case
+
+After completing your case study write-up, please submit your case document to the KubeEdge [website](https://github.com/kubeedge/website) repository, and it will be displayed in the official website's case studies page after community review.
```

Comment:
```suggestion
After completing your case study write-up, please submit your case document to the KubeEdge [website](https://github.com/kubeedge/website) repository, you can submit your case in English or Chinese(languages ​​currently supported by the website), and it will be displayed in the official website's case studies page after community review. 
```